utils/warnings.ml

```diff
@@ -426,6 +428,8 @@ let message = function
       Printf.sprintf "the %S attribute is used more than once on this expression" attr_name
   | Inlining_impossible reason ->
       Printf.sprintf "Inlining impossible in this context: %s" reason
+  | No_cmx_file name ->
+      Printf.sprintf "no cmx file was found in path for module " ^ name
```

Comment:
Would there be a way to mention in the message that `-opaque` could be used, that would not make it the Daughter of All Error Messages?